internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,57 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur", ".vscode", ".idea", ".github"}
+	err = deleteFilesRecursively(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectories(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesRecursively(deleteFiles []string, path string) error {
+	// walk through the directory
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !fileInfo.IsDir() {
+			fileName := fileInfo.Name()
 
+			for _, targetFile := range deleteFiles {
+				if fileName == targetFile {
+					filePath := filepath.Join(path, fileName)
+
+					err := os.Remove(filePath)
+					if err != nil {
+						return err
+					}
+				}
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func deleteDirectories(deleteDirs []string, path string) error {
+	for _, targetDir := range deleteDirs {
+		dirPath := filepath.Join(path, targetDir)
+		err := os.RemoveAll(dirPath)
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		if err != nil {
			if errors.Is(err, fs.ErrNotExist) {
				continue
			}
			return err
		}
```

Handle `os.ErrNotExist`. You should not return an error when a directory simply does not exist.

Reply:
// RemoveAll removes path and any children it contains.
// It removes everything it can but returns the first error
// it encounters. If the path does not exist, RemoveAll
// returns nil (no error).
// If there is an error, it will be of type *PathError.

Reply:
Oh, I did not notice that. Thanks for pointing it out!